ats/src/ats/bin/atslite3.py

```diff
@@ -80,7 +80,7 @@ def main():
             if val.isdigit():
                 numNodes = int(val)
             else:
-                print "Error '%s' is invalid" % arg
+                print("Error '%s' is invalid" % arg)
```

Comment:
Is "Error <message>" consistent across ats?  It's nice to be able to search for a consistent string like like 72's "INFO: <message>"?

Reply:
Full disclosure, there is a fair share of inconsistency lurking around the code base.
The goal of this PR is to simply take a baby step toward upgrading to the code to Python 3.

I agree with your idea though. It would be nice to make liberal use of Python's built in logging module and consolidate all logging actions into an ATS logging wrapper module of some kind.

Reply:
#21 so this doesnt get forgotten